hugegraph-api/src/main/java/com/baidu/hugegraph/api/graph/VertexAPI.java

```diff
@@ -114,6 +115,64 @@ public String create(@Context GraphManager manager,
         });
     }
 
+    /**
+     * TODO: Adapter for param("createIfNotExist") or delete it?
```

Comment:
If we really don't use this parameter, just delete it.

Reply:
@Linary any suggestion?

Reply:
delete it first, consider it when inneed